src/web/docusaurus/docs/tools-and-technologies/prisma.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 11
+---
+
+# Prisma
+
+## Our Database
+
+[Supabase](./supabase.md) is one of the technologies that Telescope uses. Because it is built on top of Postgres, we have full access to a Postgres DB that can be used to store all sorts of data used by both frontend and backend.
+
+> PostgreSQL is a highly stable database backed by more than 20 years of development by the open-source community.
+
+## Database Schema
+
+Our database schema is defined in [schema.prisma](https://github.com/Seneca-CDOT/telescope/blob/master/src/db/prisma/schema.prisma). This is where we defined models and their relations using Prisma schema syntax.
+
+Managing the database schema using Prisma, we can track granular changes to the database schema. These granular changes are reflected as separate SQL script files. That way, our granular schema changes are reflected as code that can be captured with any version control software. This makes sure that the local development database or the production database is always up to date with the latest Schema.
+
+## Database migration
+
+Whenever we start up the DB container or after updating our schema, we should run a migration to apply new changes to the database if any.
+
+## Running migration
+
+```bash
+  cd src/db
+  cp env.example .env
```

Comment:
Question: should we default to this somehow vs. making it a requirement?  That is, do we ever need to change the values in this file?  If not, maybe we should hard-code it into the script.

Reply:
Yeah, we don't have a way to run migrations on the remote DB.

Reply:
Can we hardcode the DATABASE_URL if the DB password on dev and production are different?